Seneca-CDOT/telescope
# Closes #3604: Announce Sign-Ups in Slack

## Issue This PR Addresses



## Type of Change




## Description



This PR implements the enhancement requested in #3604.

## Steps to test the PR


- Replace the constants in `env.*` files with actual credentials. For `CHANNEL_ID`, use your member id. 
- Start Telescope
- Create an account
- Check for a DM from `telescope-bot`

## Checklist




## Behaviour
<img width="1099" alt="Screenshot 2022-12-01 at 9 21 24 AM" src="https://user-images.githubusercontent.com/50856799/205077885-af4fc86a-070e-4ea1-b22e-a4147fb493cd.png">
